packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -140,156 +150,133 @@ export class Telemetry {
 
     await Telemetry.setupBaseProperties();
 
-    Telemetry.setupTelemetryProcessors();
+    // Setup telemetry initializers (known as "processors" on App Insights)
+    this.addTelemetryInitializers();
   }
 
-  /** Sets up Telemetry.client. */
-  private static setupClient() {
-    appInsights.Configuration.setInternalLogging(false, false);
+  // The first telemetry processor that is executed after calling flush().
+  // Applicable to all events.
+  // NOTE: returning true means the next processor will be executed.
+  // Otherwise, the event won't be posted.
+  private static basicTelemetryInitializer(envelope: coreOneDS.ITelemetryItem) : boolean
+  {    
+    // Filter out "legacy" events from older stable branches
+    if (envelope.name && EventNamesWeTrack.includes(envelope.name)) {
+      return true;
+    }
 
-    Telemetry.client = new appInsights.TelemetryClient(
-      Telemetry.options.setupString,
-    );
+    return false;
+  }
+
+  // The second telemetry processor that is executed after calling flush().
+  // Performs sanitization and PII removal from ExceptionData,
+  // applicable only to error scenarios.
+  private static errorTelemetryInitializer(envelope: coreOneDS.ITelemetryItem) : boolean
```

Comment:
Since we have to construct the error information with stack traces ourselves, I suspect we don't need this error processor anymore. The processors run at event submission time to fix up anything we couldn't control. When we had to use AppInsight's trackException code, it got the stack info for us, and we needed a processor to hook in and scrub it of PII.

Now that we're making our own error event data, we should just clean it up then.